Core/GDCore/Project/ObjectsContainersList.h

```diff
@@ -30,6 +30,8 @@ class GD_CORE_API ObjectsContainersList {
  public:
   virtual ~ObjectsContainersList(){};
 
+  static ObjectsContainersList MakeNewEmptyObjectsContainersList();
```

Comment:
This (MakeNewEmptyObjectsContainersList) seems unused in the codebase? You can then remove it.